src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+import { Todo } from '../../types/todo';
+import users from '../../api/users';
+
+type Props = {
+  toAdd: (toAdd: Todo) => void;
+  todos: Todo[];
+};
+
+export const TodoForm: React.FC<Props> = ({ toAdd, todos }) => {
+  const [title, setTitle] = useState('');
+  const [optionUser, setOptionUser] = useState(0);
```

Comment:
[CODE STYLE] - Avoid using one-letter variable names. Consider renaming `optionUser` to something more descriptive, like `selectedUser`.